plugins/rabbitmq/src/main/java/com/navercorp/pinpoint/plugin/rabbitmq/interceptor/RabbitMQPublishInterceptor.java

```diff
@@ -1,18 +1,26 @@
 package com.navercorp.pinpoint.plugin.rabbitmq.interceptor;
 
+import com.navercorp.pinpoint.bootstrap.config.Filter;
 import com.navercorp.pinpoint.bootstrap.context.MethodDescriptor;
 import com.navercorp.pinpoint.bootstrap.context.SpanEventRecorder;
 import com.navercorp.pinpoint.bootstrap.context.Trace;
 import com.navercorp.pinpoint.bootstrap.context.TraceContext;
+import com.navercorp.pinpoint.bootstrap.context.TraceId;
 import com.navercorp.pinpoint.bootstrap.interceptor.AroundInterceptor;
 import com.navercorp.pinpoint.bootstrap.interceptor.annotation.Scope;
 import com.navercorp.pinpoint.bootstrap.logging.PLogger;
 import com.navercorp.pinpoint.bootstrap.logging.PLoggerFactory;
+import com.navercorp.pinpoint.plugin.rabbitmq.RabbitMQClientPluginConfig;
 import com.navercorp.pinpoint.plugin.rabbitmq.RabbitMQConstants;
+import com.navercorp.pinpoint.plugin.rabbitmq.field.setter.HeadersFieldSetter;
 import com.rabbitmq.client.AMQP;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * @author Jinkai.Ma
+ * @author Jiaqi Feng
  */
 @Scope(value = RabbitMQConstants.RABBITMQ_SCOPE)
```

Comment:
Annotations have been prohibited in interceptors due to issues present in #2862 
Could you remove `@Scope` and use `InstrumentMethod.addScopedInterceptor(...)` method instead?

Reply:
Done.

Reply:
:+1: